src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,153 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> firstNode;
+    private Node<T> lastNode;
+    private int size;
+
+    private class Node<T> {
+        private T currentElement;
+        private Node<T> prevElement;
+        private Node<T> nextElement;
+
+        private Node(Node<T> prevElement, T currentElement, Node<T> nextElement) {
+            this.prevElement = prevElement;
+            this.currentElement = currentElement;
+            this.nextElement = nextElement;
+        }
+    }
+
```

Comment:
and move it to the bottom of the class 
```suggestion
    private static class Node<T>  {
        private T value;
        private Node<T> prev;
        private Node<T> next;

        private Node(Node<T> prevElement, T currentElement, Node<T> nextElement) {
            this.prev = prev;
            this.value = value;
            this.next = next;
        }
    }

```